lib/livebook/hubs/enterprise.ex

```diff
@@ -170,6 +169,10 @@ defimpl Livebook.Hubs.Provider, for: Livebook.Hubs.Enterprise do
     end
   end
 
+  def update_secret(_enterprise, _secret), do: :ok
+
+  def delete_secret(_enterprise, _secret), do: :ok
```

Comment:
Let's do `raise "not implemented"` for callbacks that are not implemented and we don't expect to be called.